### ISSUE TITLE
Is there a chance to fix the distorted texture rendering code of UW1?

### ISSUE THREAD

**Issue by SiENcE** [JohnGlassmyer/UltimaHacks#30]
Is there a chance to fix the distorted texture rendering code of UW1?

**Comment by JohnGlassmyer** [comment]
i don't understand the matter in detail, but what i gather is that more accurate texture mapping would require a lot more arithmetic per rendered frame.

how much of a guesstimate do you have as to how this would become feasible at higher processor speeds?

do you happen to know to what extent later x86 instructions, MMX i guess, would help with this?

would it make sense to be enabled as an additional detail level above "Very High"?

**Comment by JohnGlassmyer** [comment]
actually, i guess instructions like MMX aren't relevant here, the later processors that implemented them would be powerful enough to do this texture mapping without them.

**Comment by SiENcE** [comment]
I would agree with you about the speed. Is it basically possible to “inject” “more” code? If so, are there limits? Of course, you would first have to isolate the current algorithm and then write a separate test program and find a suitable solution.
How do you go about your work?

**Comment by MikeRatzlaff** [comment]
Outside the scope of this project, I'm sure, but just a question - would be it be possible to import any of the System Shock rendering code here? Would that improve things?
